install/lib/class.FroxlorInstall.php

```diff
@@ -840,6 +850,20 @@ private function _showDataForm()
 		 */
 		$section = $this->_lng['install']['serversettings'];
 		eval("\$formdata .= \"" . $this->_getTemplate("datasection") . "\";");
+		// distribution
+		if (! empty($_POST['installstep']) && $this->_data['distribution'] == '') {
+			$diststyle = 'color:red;';
+		} else {
+			$diststyle = '';
+		}
+		$formdata .= $this->_GETsectionItemCheckbox('distribution', 'jessie', ($this->_data['distribution'] == 'jessie'), $diststyle);
```

Comment:
too hardcoded, please read in xml files which distribution is supported, see https://github.com/Froxlor/Froxlor/blob/master/admin_configfiles.php#L131

Reply:
ok